Source/ASNetworkImageNode.mm

```diff
@@ -451,18 +465,26 @@ - (void)_cancelDownloadAndClearImage
   [self _cancelImageDownload];
```

Comment:
We should figure out if `_cancelDownloadAndClearImage` should happen in one transaction while holding the lock.